python/paddle/fluid/framework.py

```diff
@@ -5305,6 +5460,155 @@ def all_parameters(self):
             parameters.extend(each_block.all_parameters())
         return parameters
 
+    def state_dict(self, mode='all', scope=None):
+        """
+        Get parameters and persistable buffers of program. And set them into a dict
```

Comment:
`Get parameters and persistable buffers of program as a dict`? and the key is what? value is what?

Reply:
Done,thx.